features/upgrade/workloads/descheduler-upgrade.feature

```diff
@@ -59,20 +59,15 @@ Feature: Descheduler major upgrade should work fine
     Given I store master major version in the clipboard
     Given a pod becomes ready with labels:
       | app=descheduler |
-    Given cluster-kube-descheduler-operator channel name is stored in the :kdo_channel clipboard
-    When I run the :patch client command with:
-      | resource      | subscription                                                                   |
-      | resource_name | cluster-kube-descheduler-operator                                              |
-      | p             | [{"op": "replace", "path": "/spec/channel", "value": "<%= cb.kdo_channel %>"}] |
-      | type          | json                                                                           |
-      | n             | openshift-kube-descheduler-operator                                            |
-    Then the step should succeed
+    Given I make sure the descheduler operator gets updated successfully if needed
     And I use the "openshift-kube-descheduler-operator" project
-    Given I wait for the resource "pod" named "<%= pod.name %>" to disappear
+    Given I wait up to 180 seconds for the steps to pass:
```

Comment:
Step `status becomes :running of exactly 1 pods labeled` already contains wait logic, no need to wrap another wait.

Reply:
Done, addressed, could you please help review ? thanks !!